crypto/fipsmodule/rsa/rsa.c

```diff
@@ -448,12 +448,144 @@ int RSA_set0_crt_params(RSA *rsa, BIGNUM *dmp1, BIGNUM *dmq1, BIGNUM *iqmp) {
   return 1;
 }
 
+RSA_METHOD *RSA_meth_new(const char *name, int flags) {
+  RSA_METHOD *meth = OPENSSL_zalloc(sizeof(*meth));
+
+  if (meth != NULL) {
```

Comment:
I suggest checking for null and putting the error first in the code flow as we usually see in other functions (including the one right after).